src/shaders/colorimetry.h

```diff
@@ -96,36 +96,36 @@ struct PrimaryInfo {
 };
 
 vec3 convert_primary(vec2 xy) {
-	float X = xy.x / xy.y;
-	float Y = 1.0f;
-	float Z = (1.0f - xy.x - xy.y) / xy.y;
-	return vec3(X, Y, Z);
+    float X = xy.x / xy.y;
+    float Y = 1.0f;
+    float Z = (1.0f - xy.x - xy.y) / xy.y;
+    return vec3(X, Y, Z);
 }
 
 mat3 compute_xyz_matrix(PrimaryInfo metadata) {
-	vec3 red = convert_primary(metadata.displayPrimaryRed);
-	vec3 green = convert_primary(metadata.displayPrimaryGreen);
-	vec3 blue = convert_primary(metadata.displayPrimaryBlue);
-	vec3 white = convert_primary(metadata.whitePoint);
+    vec3 red = convert_primary(metadata.displayPrimaryRed);
+    vec3 green = convert_primary(metadata.displayPrimaryGreen);
+    vec3 blue = convert_primary(metadata.displayPrimaryBlue);
+    vec3 white = convert_primary(metadata.whitePoint);
 
-	vec3 component_scale = inverse(mat3(red, green, blue)) * white;
-	return transpose(mat3(red * component_scale.x, green * component_scale.y, blue * component_scale.z));
+    vec3 component_scale = inverse(mat3(red, green, blue)) * white;
+    return transpose(mat3(red * component_scale.x, green * component_scale.y, blue * component_scale.z));
 }
 
 const PrimaryInfo rec709_primaries = {
-    vec2(0.640f, 0.330f),     // red
-    vec2(0.300f, 0.600f),     // green
-    vec2(0.150f, 0.060f),     // blue
-    vec2(0.3127f, 0.3290f),   // whitepoint
+    vec2(0.640f, 0.330f),       // red
+    vec2(0.300f, 0.600f),       // green
+    vec2(0.150f, 0.060f),       // blue
+    vec2(0.31271f, 0.32902f),   // whitepoint
 };
 /*const*/ mat3 rec709_to_xyz = compute_xyz_matrix(rec709_primaries);
 /*const*/ mat3 xyz_to_rec709 = inverse(rec709_to_xyz);
 
 const PrimaryInfo rec2020_primaries = {
-    vec2(0.708f, 0.292f),     // red
-    vec2(0.170f, 0.797f),     // green
-    vec2(0.131f, 0.046f),     // blue
-    vec2(0.3127f, 0.3290f),   // whitepoint
+    vec2(0.708f, 0.292f),       // red
+    vec2(0.170f, 0.797f),       // green
+    vec2(0.131f, 0.046f),       // blue
+    vec2(0.31271f, 0.32902f),   // whitepoint
```

Comment:
Why did this change? These values were correct as-per https://www.itu.int/dms_pubrec/itu-r/rec/bt/R-REC-BT.2020-2-201510-I!!PDF-E.pdf before.

Reply:
oops forgot to add a comment about this
these are the actual values for Illuminant D65 (https://en.wikipedia.org/wiki/Illuminant_D65) 
the ITU always clips them

Reply:
I'd rather just use the values in the recommendation for now as that's likely what everyone will be referencing.

Reply:
done